docs/admin/ldap/configure-project-mapping.md

```diff
@@ -0,0 +1,73 @@
+# Configure LDAP/Kerberos group to project mapping
+ 
+## Introduction
+A group to project mapping allows you to add members of your LDAP group to a project without having to
+add each user manually. 
+ 
+## Prerequisites
+A server configured with LDAP or Kerberos. See [Server Configuration for Kerberos](../configure-server/#server-configuration-for-kerberos) and
```

Comment:
@kouzant and I added a new variable `ldap_group_mapping_sync_enabled` to completely disable the LDAP group mapping. By default this is set to `false`. Setting the variable to `true`  should be listed as prerequisite.